app/src/main/java/ch/epfl/culturequest/ui/profile/ProfileViewModel.java

```diff
@@ -4,16 +4,60 @@
 import androidx.lifecycle.MutableLiveData;
 import androidx.lifecycle.ViewModel;
 
+import java.util.List;
+
+import ch.epfl.culturequest.database.Database;
+import ch.epfl.culturequest.social.Image;
+import ch.epfl.culturequest.social.Profile;
+import ch.epfl.culturequest.utils.EspressoIdlingResource;
+
+
 public class ProfileViewModel extends ViewModel {
 
     private final MutableLiveData<String> mText;
+    private final MutableLiveData<String> profilePictureUri;
+
+    private final MutableLiveData<List<Image>> pictures;
 
     public ProfileViewModel() {
         mText = new MutableLiveData<>();
-        mText.setValue("This is profile fragment");
+        profilePictureUri = new MutableLiveData<>();
+        pictures = new MutableLiveData<>();
+        Database db = new Database();
+        EspressoIdlingResource.increment();
+        db.getProfile("123").whenComplete((p, e) -> {
```

Comment:
is this supposed to be here?

Reply:
If you're talking of the use of the Database of course this is just temporary until we are able to connect it with the active user